server/src/test/java/org/apache/iotdb/db/engine/storagegroup/DataRegionTest.java

```diff
@@ -1111,7 +1111,7 @@ public void testDeleteDataInUnSeqFlushingMemtable()
     // delete data which is in work memtable
     dataRegion.deleteByDevice(new PartialPath("root.vehicle.d0.s0"), 50, 100, 0, null);
     dataRegion.deleteByDevice(new PartialPath("root.vehicle.d0.s0"), 50, 150, 0, null);
-    dataRegion.deleteByDevice(new PartialPath("root.vehicle.d0.s0"), 100, 300, 0, null);
+    dataRegion.deleteByDevice(new PartialPath("root.vehicle.d0.s0"), 100, 190, 0, null);
```

Comment:
Why do you change this?

Reply:
Before this pr, there is a bug about updating last flush time for a empty memtable. And if the bug has been fixed, this unit test needs to be corrected to prevent error...